src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1292,784 +1308,797 @@ void MulticopterPositionControl::control_auto(float dt)
 }
 
 void
-MulticopterPositionControl::task_main()
+MulticopterPositionControl::update_velocity_derivative()
```

Comment:
I am just wondering if it is not better to keep function more general. for instance, if I also would like to compute derivative of acceleration, or the derivative of the velocity component parallel to path (which I do in my code), then we need to write a new function for all of them. 
we could write a function that updates the derivative of any derivative block